go/tools/builders/nogo_main.go

```diff
@@ -396,7 +396,7 @@ func load(packagePath string, imp *importer, filenames []string) (*goPackage, er
 	}
 	pkg.types, pkg.typesInfo = types, info
 
-	pkg.facts, err = facts.NewDecoder(pkg.types).Decode(imp.readFacts)
+	pkg.facts, err = facts.NewDecoder(pkg.types).Decode(true, imp.readFacts)
```

Comment:
```suggestion
	pkg.facts, err = facts.NewDecoder(pkg.types).Decode(true  /* skipMethodSorting */, imp.readFacts)
```

Similar to above. Naked parameters in function calls can hurt readability.

Reply:
Added